File: 2021/ProblemA.py
word = "chi"
user_letter = input("Enter a letter: ").lower().strip()
if len(user_letter) == 1 and user_letter.isalpha():
    if user_letter in word:
        print("Yes")
    else:
        print("No")
elif len(user_letter) == user_letter.isalnum():
    print("Enter a letter and not a number!")
else:
    print("Please, insert a single letter.")

File: 2021/ProblemA.java
import java.util.Scanner;

public class ProblemA {
    public static void main(String[] args) {
        String word = "chi";
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter a letter: ");
        String userLetter = scanner.nextLine().toLowerCase().trim();

        if (userLetter.length() == 1 && Character.isLetter(userLetter.charAt(0))) {
            if (word.contains(userLetter)) {
                System.out.println("Yes");
            }
            else {
                System.out.println("No");
            }
        }
        else {
            System.out.println("Please, insert a single letter.");
        }

        scanner.close();
    }
}